libs/webb-ui-components/src/components/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -36,12 +35,17 @@ export const BreadcrumbsItem = React.forwardRef<
     return (
       <Chip
         color="grey"
```

Comment:
The default props should be defined in the component (`isLast` here), instead of using `BreadcrumbsItem.defaultProps = BreadcrumbsItemsDefaultProps;` as you did. You can read more about this [here](https://react.dev/learn/passing-props-to-a-component#specifying-a-default-value-for-a-prop).